weekly-updates/project-lead-meetings/08_05_2018.md

```diff
@@ -136,15 +136,28 @@ _Proposed Deadline_:
 ## snackru
 
 _UPDATES_:
+LCS login works and it checks LCS for mentor role
 
 _QUESTIONS_:
+can you make amulya an org member
+LCS /auth takes 6 secs on my machine?!?!?!
 
 _TO DO_:
+upgrade meteor
+start writing tests
+get ci integration
+implement stackexchange integration
+implement floorplan pinning
 
 _Wiki Status_:
+no wiki??
 
 _How is your team running?  How do you delegate tasks? Need any assistance?_:
+great, an architect missed a meeting and I still need to meet up with them but I think that's kinda my fault because I should have put a reminder earlier in the day.
 
 _Feedback for RND Team_:
+what happened to LCS the other day? wasn't there ci/cd?
```

Comment:
It was a deploy issue. So jackass decided to start the cffi library name with a dot, so the deploy script didn't copy it since it was a hidden file.

There were other minor issues like the EC2 python3 being 3.7 while the lambda one being 3.6, but the essential fix was to grab those hidden files.

We'll look at CI for this too, but it's a bit difficult to set up. Much reading and learning must be done.